src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int randomIndex = new Random().nextInt(Color.values().length);
```

Comment:
It's bad practice to initialize dependencies in the method. You can initialize it in fields, but good practice initialization using a constructor. So, please, move Random instance initialization from this method.